### ISSUE TITLE
not support nodejs v14

### ISSUE THREAD

**Issue by gaopanfeng** [damirn/rapidx2j#43]
npm install with v14.15.0

```
../src/parser.cc:100:41: error: no matching member function for call to 'Set'
      v8::Local<v8::Object>::Cast(ret)->Set(Nan::New<v8::String>(options.attributePrefix).ToLocalChecked(), attr);
```

**Comment by damirn** [comment]
Hey, thanks for reporting this one. Will work on it ASAP.

**Comment by damirn** [comment]
Should be fixed with v0.6.0.
